File: Java/프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.java
class Solution {
    public int[] solution(int n) {
        int d = 0;
        int num = (n+1)/2;
        int[] answer = new int[num];
    
            for(int i=0; i<num; i++) {
                answer[i] = (i*2)+1;
            }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.js
function solution(n) {
    var answer = [];
    if(n % 2 != 0)
        answer.push(n);
    
    while(n > 1){
        if(n % 2 == 0){ //짝수이면
            n = n-1;
            answer.push(n);
        } else{ //홀수이면
            n = n-2;
            answer.push(n);
        }
    }
    answer.sort((a, b) => a-b);
    return answer;
}